examples/api/test/material/color_scheme/dynamic_content_color.0_test.dart

```diff
@@ -0,0 +1,156 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_api_samples/material/color_scheme/dynamic_content_color.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+
+  testWidgets('The theme colors are created dynamically from the first image', (WidgetTester tester) async {
+    await HttpOverrides.runZoned(
+      () async {
+        await tester.pumpWidget(
+          const example.DynamicColorExample(),
+        );
+
+        expect(
+          find.widgetWithText(AppBar, 'Content Based Dynamic Color'),
+          findsOne,
+        );
+        expect(find.byType(Switch), findsOne);
+        expect(find.byIcon(Icons.light_mode), findsOne);
+
+        // Loads the images.
```

Comment:
```suggestion
        // Loads the images.
        // Using runAsync forces the streams to complete. This is needed because loading the fake image
        // is a real async task.
```

Here is a proposed comment to give a clue about why the `runAsync` are needed.

Reply:
Nice, I've updated it in [format: Add better comment for the runAsyncs and better formatting](https://github.com/flutter/flutter/pull/158309/commits/90d28317d442abbfa4cb8c36f904a2d63f42a47b) 😊